sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkSQLCLIDriver.scala

```diff
@@ -539,6 +540,10 @@ private[hive] class SparkSQLCLIDriver extends CliDriver with Logging {
       index > beginIndex && !s"${line.charAt(index)}".trim.isEmpty)
```

Comment:
How about `statementBegin` -> `statementInProgress` and `includingStatement` -> `isStatement`?

Reply:
ok